README.md

```diff
@@ -23,9 +23,9 @@ Ensure you have service-manager python environment setup:
 
 `source ../servicemanager/bin/activate`
 
-`sm --start DC_TWSM_ALL`
+`sm2 --start DC_TWSM_ALL`
 
-`sm --stop SECURE_MESSAGE`
+`sm2 --stop SECURE_MESSAGE`
```

Comment:
Minor point, but there was a pattern to naming these profiles - starting DC_<your service name>_IT. Might be nice to keep them consistent